kodilla-basic-tests/src/test/java/com/kodilla/basic_asseration/CalculatorTestSuite.java

```diff
@@ -0,0 +1,60 @@
+package com.kodilla.basic_asseration;
+
+import com.kodilla.basic_assertion.Calculator;
+import org.junit.Test;
+
+import static com.kodilla.basic_assertion.ResultChecker.assertEquals;
+
+
+public class CalculatorTestSuite {
+    Calculator calculator =new Calculator();
+
+
+    @Test
+    public void testSum() {
+        //Given
+        int a = 5;
+        int b = 8;
+        //When
+        int sumResult = calculator.sum(a, b);
+        //Then
+        assertEquals(13, sumResult);
+    }
+
+    @Test
+    public void testSubtract() {
+
+        int a = 10;
+        int b = 7;
+        int subtractResult = calculator.subtract(a, b);
+        assertEquals(3, subtractResult);
```

Comment:
uzywaj prosze konwencji //Given //When //Then
lub //Arrange //Act //Assert

przyklad np tutaj
https://stackoverflow.com/questions/33847470/given-when-then-convention-together-with-exception-handling-with-mockito-and